crates/js-sys/src/lib.rs

```diff
@@ -3679,6 +3710,12 @@ impl From<String> for JsString {
     }
 }
 
+impl From<char> for JsString {
+    fn from(c: char) -> Self {
```

Comment:
Should probably be `#[inline]`

Reply:
I'm usually strictly against using `#[inline]` as an anti-pattern, but given how widespread it is in the codebase already, I guess doesn't harm to add here too.

Reply:
Done.

Reply:
Why do you think it's an anti-pattern? What's wrong with improving performance in non-LTO builds? Keep in mind that LTO is disabled by default even in release mode.

Obviously inlining can be overused, but in the case where the function/method is known to be small (like in this case), why not?